Sources/TensorFlow/Layers/Dropout.swift

```diff
@@ -73,3 +73,68 @@ public struct Dropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
     }
   }
 }
+
+/// A gaussian noise layer
+///
+/// Gaussian Noise adds noise sampled from a normal distribution with mean zero
+public struct GaussianNoise<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
+  @noDerivative public let standardDeviation: Tensor<Scalar>
```

Comment:
It looks like we're being inconsistent between whether we store `Tensor<Scalar>` or `Double` above. I think the most conservative thing to do is to keep this as a `Scalar` instance property.

Reply:
I'm not sure I've understood this @saeta ? Are you suggesting to remove `TensorFlowFloatingPoint` ? The layer seems consistent in scalar otherwise. The dropout implementation has the `Double`/`Scalar` confusion in it's implementation.

Reply:
I think I was suggesting:

```suggestion
  @noDerivative public let standardDeviation: Scalar
```